moviedetails/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    compileSdk 32
+
+    defaultConfig {
+        minSdk 21
+        targetSdk 32
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
```

Comment:
I suggest this line of code is implemented in this way:

`  jvmTarget = JavaVersion.VERSION_1_8.toString()`

Reply:
Ok, got it Uriel

Reply:
Like this?
![Screen Shot 2022-05-03 at 16 58 47](https://user-images.githubusercontent.com/104175612/166573386-8862d3a5-7ae7-4fca-9c86-2997b2dc2d67.png)